### ISSUE TITLE
Big lighthouse score impact on web

### ISSUE THREAD

**Issue by natew** [th3rdwave/react-native-safe-area-context#293]
Finally narrowed this down in my app - this package triggers reflows on mount on the web. I haven't taken the time to track it down yet but will revisit it soon, but figure it's worth tracking.

**Comment by jacobp100** [comment]
Yeah - something like `calc(env(...) + someProp)` would probably work better. See this file - https://github.com/th3rdwave/react-native-safe-area-context/blob/main/src/SafeAreaView.web.tsx#L55

**Comment by jacobp100** [comment]
Happy to take a look at PRs and give pointers
